services/fee-estimator/shared/utils/transactionsUtils.js

```diff
@@ -18,44 +18,44 @@ const { computeMinFee } = require('@liskhq/lisk-transactions');
 const { requestConnector } = require('./request');
 const { getGenesisConfig } = require('../networkConstants');
 
-let allTransactionSchemasCache;
+let allCommandsParamsSchemasCache;
 
-const getTransactionsSchemasFromCore = async () => {
-	const schemas = await requestConnector('getSchema', {});
+const getCommandsParamsSchemasFromCore = async () => {
+	const schemas = await requestConnector('getSchema');
 	return schemas;
 };
 
-const getAllTransactionSchemasFromCache = async () => {
-	if (!allTransactionSchemasCache) {
-		allTransactionSchemasCache = await getTransactionsSchemasFromCore();
+const getAllCommandsParamsSchemasFromCache = async () => {
+	if (!allCommandsParamsSchemasCache) {
+		allCommandsParamsSchemasCache = await getCommandsParamsSchemasFromCore();
 	}
-	return allTransactionSchemasCache;
+	return allCommandsParamsSchemasCache;
 };
 
-const getTxnAssetSchema = async (trx) => {
-	const moduleAssetId = String(trx.moduleID).concat(':').concat(trx.assetID);
-	const response = await getAllTransactionSchemasFromCache();
-	const allTransactionSchemas = response.transactionsAssets.map(txAsset => {
-		const formattedTxAsset = {};
-		formattedTxAsset.moduleAssetId = String(txAsset.moduleID).concat(':').concat(txAsset.assetID);
-		formattedTxAsset.moduleAssetName = String(txAsset.moduleName).concat(':').concat(txAsset.assetName);
-		formattedTxAsset.schema = txAsset.schema;
-		return formattedTxAsset;
+const getTxnParamsSchema = async (trx) => {
+	const moduleCommandID = String(trx.moduleID).concat(':').concat(trx.commandID);
+	const response = await getAllCommandsParamsSchemasFromCache();
+	const allCommandsParamsSchemas = response.commands.map(txParams => {
+		const formattedTxParams = {};
+		formattedTxParams.moduleCommandID = String(txParams.moduleID).concat(':').concat(txParams.commandID);
+		formattedTxParams.moduleCommandName = String(txParams.moduleName).concat(':').concat(txParams.commandName);
+		formattedTxParams.schema = txParams.schema;
+		return formattedTxParams;
 	});
```

Comment:
If `getTxnParamsSchema` is invoked very frequently, please consider caching `allCommandsParamsSchemas` to avoid unnecessary repetition of the logic.